content/en/blog/_posts/2023-08-15-retroactive-default-storage-class-ga.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: blog
+title: "Kubernetes v1.28: Retroactive Default StorageClass move to GA"
+date: 2023-08-15
+slug: retroactive-default-storage-class-ga
+---
+
+**Author:** Roman Bednář (Red Hat)
+
+Announcing graduation to General Availability (GA) - Retroactive Default StorageClass Assignment in Kubernetes v1.28!
+
+We are thrilled to announce that the "Retroactive Default StorageClass Assignment" feature,
```

Comment:
Who are “we” here? Kubernetes SIG Storage perhaps?

Reply:
I meant Kubernetes community but it's not too apparent for the reader. Since there are links to SIG Storage below we can use Kubernetes SIG storage here instead.